.github/workflows/ci.yml

```diff
@@ -103,6 +103,21 @@ jobs:
       - name: Build and Test
         run: python ./build.py --no-check
 
+  format-qsc:
```

Comment:
Sorry, noticed one more thing! This is purely about aesthetics again, but this should live just below the other `Format` job so that the CI output looks more organized.

![image](https://github.com/microsoft/qsharp/assets/16928427/af42d6d0-7ded-49c8-a6e2-725848de88ca)

Reply:
I'm not entirely sure how this all works, but the reason I put it there was because the formatter needs to be built, so I put it after the build steps. Does that not have any impact?

Reply:
Oh no these jobs all run in parallel on different agents. I think it's just `cargo run` building your code.

Reply:
done